ui/src/app/completed/services/tableService.spec.js

```diff
@@ -0,0 +1,35 @@
+describe('Service: TableService', function() {
```

Comment:
This is fixed with [9a50ccb](https://github.com/scorelab/Bassa/pull/104/commits/9a50ccbfbea3d69c5dbe0945a90126a18a1ff5c1)